sdk/src/main/java/io/dapr/client/DaprClientGrpc.java

```diff
@@ -382,14 +386,19 @@
    *
    * @param httpExtension Object for HttpExtension
    * @param appId         The application id to be invoked
+   * @param contentType   The content type of the request body
    * @param method        The application method to be invoked
    * @param request       The body of the request to be send as part of the invocation
    * @param <K>           The Type of the Body
    * @return The object to be sent as part of the invocation.
    * @throws IOException If there's an issue serializing the request.
    */
   private <K> DaprProtos.InvokeServiceRequest buildInvokeServiceRequest(
```

Comment:
This method invokes objectSerializer, so it can get the contenttype via objectSerializer.getContentType().

Reply:
Removed it.